Snakefile

```diff
@@ -150,9 +156,24 @@ rule merge_input:
     output: dataset_file = SEP.join([out_dir, '{dataset}-merged.pickle'])
     run:
         # Pass the dataset to PRRunner where the files will be merged and written to disk (i.e. pickled)
-        dataset_dict = get_dataset(_config.config.datasets, wildcards.dataset)
+        dataset_dict = get_dict(_config.config.datasets, wildcards.dataset)
         runner.merge_input(dataset_dict, output.dataset_file)
 
+# Return all files used in the gold standard
+def get_goldstandard_dependencies(wildcards):
```

Comment:
Sometimes functions and variables names have `_` between `gold` and `standard` and sometimes they don't. Consistency may help avoid future mistakes.